filechooser-portal/Main.vala

```diff
@@ -38,6 +38,22 @@ public class Files.FileChooserPortal : Object {
         dialogs = new HashTable<string, FileChooserDialog> (str_hash, str_equal);
     }
 
+    /** open_file: (async):
+     * @handle: Object path for the portal.Request object representing this call
+     * @app_id: Application originating the call (not used)
+     * @parent_window: parent window information
+     * @title: title for the file chooser dialog
+     * @options: "modal", "current-name", "current-folder", "current-file",
+                 "filters", "current-filter", "choices"
+     * @response: (out): the user response
+     * @results: (out): URI to open, choices and read-only information
+
+     * Asynchronously requests the user for a URI to open.
+     * If the URI exists, asks the user for permission to overwrite. If the user
+     * agrees, then the URI is considered safe and added to @results
+     *
+     * If it does not exist, the URI is implicitly considered safe.
+     */
```

Comment:
AFAIK, Valadoc style don't include the method name on top, and specify the parameters after the description, Also `@param param_name description` is used instead of `@param_name: description`.

```suggestion
    /**
     * Requests the user for a URI.
     *
     * The URI should point to a file, if "multiple" is true, more than one URI
     * can be chosen by the user. If "directory" is true, the URI should point
     * to a folder instead.
     *
     * A filter can be specified with "current_filter", or "filters", if more
     * than one filter could be used. Application specific options can be added
     * to the dialog with "choices" and a label to the select button set with
     * "accept_label".
     *
     * @param handle 
     *    Object path where the request should be exported
     * @param app_id 
     *    Application originating the call (not used)
     * @param parent_window 
     *    Transient parent handle, in format "type:handle". if type is "x11",
     *    handle should be a XID in hexadecimal format, if type is "wayland",
     *    handle should be a xdg_foreign handle. Other types aren't supported.
     * @param title 
     *    title for the file chooser dialog
     * @param options 
     *    Dictionary of extra options. valid keys are: "accept_label",
     *    "choices", "current_filter", "directory", "filters", "multiple"
     *    and "modal".
     * @param response 
     *    User response, 0 on success, 1 if cancelled, 2 on fail.
     * @param results 
     *    Dictionary with the user choices, possible keys are: "uris",
     *    "choices", "writable" and "current_filter".
     */
```